COMMITS.md

```diff
@@ -16,10 +16,18 @@ Use this git hook to auto-check your commit messages. Save the following snippet
 
 ```
 #!/bin/sh
+
+commit_msg=$(cat "$1")
+if echo "$commit_msg" | grep -qE "^fixup! "; then
+  # Skip validation in case of fixup commit
+  exit 0
+fi
+
 if ! grep -qE "^(build|ci|docs|feat|fix|perf|refactor|style|test|chore|revert)(\((blockchain-link|components|components-storybook|integration-tests|suite|suite-data|suite-desktop|suite-native|suite-onboarding|suite-storage|suite-web)\))?: " "$1" ; then
```

Comment:
something like this?
```suggestion
if ! grep -qE "^(build|ci|docs|feat|fix|perf|refactor|style|test|chore|revert)(\([a-z]+\))?: " "$1" ; then
```
to allow 
`feat: anything`
`feat(foo): bar`

but not allow
`bla bal`
`feat bla`
`feat(232NOOO): bla`
`feat(a)`
...

Reply:
Sounds good, I just added `-` character